tests/page/page-route.spec.ts

```diff
@@ -672,3 +672,13 @@ it('should support the times parameter with route matching', async ({ page, serv
   await page.goto(server.EMPTY_PAGE);
   expect(intercepted).toHaveLength(1);
 });
+
+it('should contain raw header', async ({ page, server }) => {
+  let headers: any;
+  await page.route('**/*', async route => {
+    headers = await route.request().allHeaders();
```

Comment:
Check that it actually returns expected headers?

Reply:
I don't think we are returning raw headers here, so I only check that it `accept` one exists.